<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainView</class>
 <widget class="QMainWindow" name="MainView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CuteClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="filterBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>350</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>700</height>
         </size>
        </property>
        <property name="title">
         <string>Filters</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="filterLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>10</number>
             </property>
             <property name="leftMargin">
              <number>15</number>
             </property>
             <property name="topMargin">
              <number>15</number>
             </property>
             <property name="rightMargin">
              <number>15</number>
             </property>
             <property name="bottomMargin">
              <number>15</number>
             </property>
             <item>
              <widget class="QLabel" name="dateLabel">
               <property name="text">
                <string>Date</string>
               </property>
               <property name="buddy">
                <cstring>dateFilter</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="dateFilter"/>
             </item>
             <item>
              <widget class="QLabel" name="idLabel">
               <property name="text">
                <string>ExternalID</string>
               </property>
               <property name="buddy">
                <cstring>idFilter</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="idFilter"/>
             </item>
             <item>
              <widget class="QLabel" name="numLabel">
               <property name="text">
                <string>Number</string>
               </property>
               <property name="buddy">
                <cstring>numberFilter</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="numberFilter"/>
             </item>
             <item>
              <widget class="QLabel" name="constLabel">
               <property name="text">
                <string>Constant</string>
               </property>
               <property name="buddy">
                <cstring>constantFilter</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="constantFilter"/>
             </item>
             <item>
              <widget class="QLabel" name="digLabel">
               <property name="text">
                <string>Digits</string>
               </property>
               <property name="buddy">
                <cstring>digitsFilter</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="digitsFilter"/>
             </item>
             <item>
              <widget class="QLabel" name="decLabel">
               <property name="text">
                <string>Decimals</string>
               </property>
               <property name="buddy">
                <cstring>decimalsFilter</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="decimalsFilter">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="searchBtn">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="layoutDirection">
                <enum>Qt::RightToLeft</enum>
               </property>
               <property name="text">
                <string>Search</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="listBox">
        <property name="title">
         <string>    Date                    ExternalID                                    Number               Constant        Digits              Decimals</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QListView" name="listView">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>350</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>dateFilter</tabstop>
  <tabstop>idFilter</tabstop>
  <tabstop>numberFilter</tabstop>
  <tabstop>constantFilter</tabstop>
  <tabstop>digitsFilter</tabstop>
  <tabstop>decimalsFilter</tabstop>
  <tabstop>searchBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
